README.md

```diff
@@ -2,6 +2,30 @@
 
 Precompile-EVM is a repository for registering precompiles to Subnet-EVM without forking the Subnet-EVM codebase. Subnet-EVM supports registering external precompiles through `precompile/modules` package. By importing Subnet-EVM as a library, you can register your own precompiles to Subnet-EVM and build it together with Subnet-EVM.
 
+## Environment Setup
+
+To effectively build, run and test a Precompile-EVM, you need software such as golang, node,[AvalancheGo](https://github.com/ava-labs/avalanchego), and [Avalanche Network Runner](https://github.com/ava-labs/avalanche-network-runner). 
+
+To get started easily, we provide a Dev Container specification, that can be used using GitHub Codespace or locally using Docker and VS Code. DevContainers are a concept that utilizes containerization (via Docker containers) to create consistent and isolated development environment. We can access this environment through VS code, so it will not feel very different from the local development.
+
+### Dev Container in Codespace
+
+Codespaces is a development environment service offered by GitHub that allows developers to write, run, test, and debug their code directly on a cloud machine provided by GitHub. The developer can edit the code through a VS Code running in the browser or locally.
+
+To run a Codespace click on the **Code** and switch to the **Codespaces** tab. There click **Create Codespace on branch [...]**. 
+
+### Local Dev Container
+
+In order to run the Dev Container locally:
+
+- Install VS Code, Docker and the [Dev Container Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+- Clone the Repository
+- Open the Container by issuing the Command "Dev Containers: Reopen in Container" in the VS Code command palette (Cmd + Shift + P).
+
+## Learn about Precompile-EVM
+
+To get a comprehensive introduction to Precompile-EVM, take the Avalanche Academy course [Customize the EVM](https://academy.avax.com/course/customize-evm).
```

Comment:
"To get a comprehensive introduction to Precompile-EVM, take the Avalanche Academy course on [Customizing the EVM](https://academy.avax.com/course/customize-evm)."